### DISCUSSION TITLE
BdBrowser Userscirpt

### DISCUSSION THREAD

**Discussion by T3M1N4L** [tsukasa/BdBrowser#40] [Ideas]
I need a userscript version of betterdiscord for a project, and I know it's possible to turn extensions into userscripts by sacraficing some features (That use extension APIs). I can use vencord, but vencord has it's own problems.

**Comment by draepiction** [comment]
i second this

> I need a userscript version of betterdiscord for a project, and I know it's possible to turn extensions into userscripts by sacraficing some features (That use extension APIs). I can use vencord, but vencord has it's own problems.
